lazy_tensor_core/lazy_tensor_core/csrc/lazy_graph_executor.cpp

```diff
@@ -9,19 +9,20 @@
 
 // TODO: DebugUtil will be upstreamed after LazyTensor is in.
 //#include "lazy_tensor_core/csrc/debug_util.h"
+#include <torch/csrc/lazy/core/internal_ops/arithmetic_ir_ops.h>
```

Comment:
what does 'internal_ops' mean really?

i wonder if it would be better to start by putting these ops under lazy/ts_backend/ops/

later when (if) we move certain ops to core folder, i'm still not sure what internal means.  is it to distinguish ops that are not part of aten schema?  or where will the ops that are part of aten but hand-coded instead of codegen live?

Reply:
Yes, by internal_ops, I mean ops that are not coming from aten, such as Scalar, DeviceData, etc..  You are right that arithmetic ops probably should not belong this category and their implementation can be changed.

I will follow you suggestion to move all the ops here under ts_backend.